engine/model_transaction_config_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 var (
-	emptyConfigJSON = "{\"change_destinations\":[{\"created_at\":\"0001-01-01T00:00:00Z\",\"updated_at\":\"0001-01-01T00:00:00Z\",\"deleted_at\":null,\"id\":\"c775e7b757ede630cd0aa1113bd102661ab38829ca52a6422ab782862f268646\",\"xpub_id\":\"1a0b10d4eda0636aae1709e7e7080485a4d99af3ca2962c6e677cf5b53d8ab8c\",\"locking_script\":\"76a9147ff514e6ae3deb46e6644caac5cdd0bf2388906588ac\",\"type\":\"pubkeyhash\",\"chain\":1,\"num\":123,\"paymail_external_derivation_num\":null,\"address\":\"1CfaQw9udYNPccssFJFZ94DN8MqNZm9nGt\",\"draft_id\":\"test-reference\",\"derivation_method\":\"\",\"sender_xpub\":\"\",\"output_index\":0}],\"change_destinations_strategy\":\"\",\"change_minimum_satoshis\":0,\"change_number_of_destinations\":0,\"change_satoshis\":124,\"expires_in\":20000000000,\"fee\":12,\"fee_unit\":{\"satoshis\":1,\"bytes\":20},\"from_utxos\":null,\"include_utxos\":null,\"inputs\":null,\"outputs\":null,\"sync\":null}"
+	emptyConfigJSON = "{\"change_destinations\":[{\"created_at\":\"0001-01-01T00:00:00Z\",\"updated_at\":\"0001-01-01T00:00:00Z\",\"deleted_at\":null,\"id\":\"c775e7b757ede630cd0aa1113bd102661ab38829ca52a6422ab782862f268646\",\"xpub_id\":\"1a0b10d4eda0636aae1709e7e7080485a4d99af3ca2962c6e677cf5b53d8ab8c\",\"locking_script\":\"76a9147ff514e6ae3deb46e6644caac5cdd0bf2388906588ac\",\"type\":\"pubkeyhash\",\"chain\":1,\"num\":123,\"paymail_external_derivation_num\":null,\"address\":\"1CfaQw9udYNPccssFJFZ94DN8MqNZm9nGt\",\"draft_id\":\"test-reference\",\"derivation_method\":\"\",\"sender_xpub\":\"\",\"output_index\":0}],\"change_destinations_strategy\":\"\",\"change_minimum_satoshis\":0,\"change_number_of_destinations\":0,\"change_satoshis\":124,\"expires_in\":20000000000,\"fee\":12,\"fee_unit\":{\"satoshis\":1,\"bytes\":1000},\"from_utxos\":null,\"include_utxos\":null,\"inputs\":null,\"outputs\":null,\"sync\":null}"
```

Comment:
just thinking would not be better to create struct to represent this json string ?
```go
// ChangeDestination represents each change destination entry.
type ChangeDestination struct {
	CreatedAt                    time.Time  `json:"created_at"`
	UpdatedAt                    time.Time  `json:"updated_at"`
	DeletedAt                    *time.Time `json:"deleted_at"` // Pointer to handle null values
	ID                           string     `json:"id"`
	XpubID                       string     `json:"xpub_id"`
	LockingScript                string     `json:"locking_script"`
	Type                         string     `json:"type"`
	Chain                        int        `json:"chain"`
	Num                          int        `json:"num"`
	PaymailExternalDerivationNum *int       `json:"paymail_external_derivation_num"` // Pointer to handle null values
	Address                      string     `json:"address"`
	DraftID                      string     `json:"draft_id"`
	DerivationMethod             string     `json:"derivation_method"`
	SenderXpub                   string     `json:"sender_xpub"`
	OutputIndex                  int        `json:"output_index"`
}

// FeeUnit represents the fee unit structure.
type FeeUnit struct {
	Satoshis int `json:"satoshis"`
	Bytes    int `json:"bytes"`
}

// Request represents the full JSON structure.
type Request struct {
	ChangeDestinations         []ChangeDestination `json:"change_destinations"`
	ChangeDestinationsStrategy string              `json:"change_destinations_strategy"`
	ChangeMinimumSatoshis      int                 `json:"change_minimum_satoshis"`
	ChangeNumberOfDestinations int                 `json:"change_number_of_destinations"`
	ChangeSatoshis             int                 `json:"change_satoshis"`
	ExpiresIn                  int64               `json:"expires_in"`
	Fee                        int                 `json:"fee"`
	FeeUnit                    FeeUnit             `json:"fee_unit"`
	FromUtxos                  interface{}         `json:"from_utxos"`  // Using interface{} for null
	IncludeUtxos               interface{}         `json:"include_utxos"`  // Using interface{} for null
	Inputs                     interface{}         `json:"inputs"`  // Using interface{} for null
	Outputs                    interface{}         `json:"outputs"`  // Using interface{} for null
	Sync                       interface{}         `json:"sync"`  // Using interface{} for null
}
```
what u think ?